GiniCapture/Classes/Core/Screens/Camera/CameraPreviewViewController.swift

```diff
@@ -198,7 +197,9 @@ final class CameraPreviewViewController: UIViewController {
     
     func stopLoadingIndicator(){
         DispatchQueue.main.async {
-            self.spinner.stopAnimating()
+            if (self.spinner != nil){
```

Comment:
This made me think 🤔. Wouldn't it be better to create the spinner in a lazy computed property like the `previewView` and add it in `loadView()`? Then it can never be `nil` and you can simply add it as a subview without needing to dispatch to the main queue.

Reply:
Done . Good point👍🏻